doc/topics/packaging/index.rst

```diff
@@ -10,10 +10,78 @@ Relenv onedir packaging
 Starting in 3006, only onedir packaging will be available. The 3006 onedir packages
 are built with the `relenv <https://github.com/saltstack/relative-environment-for-python>`_ tool.
 
+
+Docker Containers
+=================
+We use docker containers to build our packages. If you are building your own packages you can use
+the same containers we build with in the Github piplines. These containers are documented `here <https://github.com/saltstack/salt-ci-containers/tree/main/custom/packaging>`_.
+
+
+How to build onedir only
+========================
+
+Install relenv:
+
+.. code-block:: bash
+
+   # pip install relenv
+
+Fetch toolchain (Only required for linux OSs)
+
+.. code-block:: bash
+
+   # relenv toolchain fetch --arch=<architecture>
+
+Fetch Native Python Build:
```

Comment:
I would remove `--arch` from these commands. Cross compiling is still experimental in relenv and relenv will default to the native architecture.